src/libraries/System.Transactions.Local/tests/OleTxTests.cs

```diff
@@ -529,6 +529,7 @@ public void ImplicitDistributedTransactions_cannot_be_changed_after_being_set()
     }
 
     [ConditionalFact(nameof(IsRemoteExecutorSupportedAndNotNano))]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/77241")]
     public void ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true()
```

Comment:
Should the `ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_false` right below this also be disabled?

Reply:
I'm not seeing a test failure for that in #77241, did I miss another failure somewhere?

I agree that the failure likely isn't specifically related to ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true, and part of the reason for disabling it is to see if the failure slips to another test, which may provide a bit more information... Any thoughts?

Reply:
I can't find any failures for that one either so that sounds good to me.